components/echo-server/main_test.py

```diff
@@ -15,68 +17,57 @@
 import base64
 import json
 import os
-
 import pytest
 
 import main
 
 
 @pytest.fixture
 def client(monkeypatch):
-    monkeypatch.chdir(os.path.dirname(main.__file__))
-    main.app.testing = True
-    client = main.app.test_client()
-    return client
+  monkeypatch.chdir(os.path.dirname(main.__file__))
+  main.app.testing = True
+  client = main.app.test_client()
+  return client
 
 
 def test_echo(client):
-    r = client.post(
-        '/echo',
-        data='{"message": "Hello"}',
-        headers={
-            'Content-Type': 'application/json'
-        })
+  r = client.post(
+    '/echo',
```

Comment:
This looks a little weird; I would have expected

r = client.post('/echo',
                       data=....
..

Reply:
Hmm, the recent version I have it looks like this:
```
def test_echo(client):
  r = client.post(
      '/echo',
      data='{"message": "Hello"}',
      headers={'Content-Type': 'application/json'})

  assert r.status_code == 200
  data = json.loads(r.data.decode('utf-8'))
  assert data['message'] == 'Hello'
```
which looks normal because it's continuation of line.